lib/create-release-action.js

```diff
@@ -21,16 +21,11 @@ jobs:
     name: release
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           node-version: "lts/*"
-      - uses: actions/cache@v1
-        with:
-          path: ~/.npm
-          key: \${{ runner.os }}-node-\${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            \${{ runner.os }}-node-
```

Comment:
Do you mind explaining a little more about the reason for the above caching changes? Presumably it's due to the 2 major version upgrade, though I couldn't find an obvious culprit in the recommended [README](https://github.com/actions/cache/blob/main/examples.md#node---npm) [diffs](https://github.com/actions/cache/blob/v1/examples.md#node---npm).

Reply:
This is the source of the changes
https://github.com/octokit/openapi/commit/0420264dc65e564d72e5efba4031a5ab1c69ccad

`actions/cache` was dropped in favour of using the cache in `actions/setup-node`